### PR TITLE
rspamd: disable checks for authenticated users

### PR DESCRIPTION
**Pull request by georglauterbach**
# Description

Disable all checks for authenticated users. See <https://github.com/docker-mailserver/docker-mailserver/issues/3323#issuecomment-1645978207>.

The reason my PR deviates from the changes proposed by @williamdes is that enabling SPF, DKIM and policy checks for _outbound_ emails still does not make sense really (because DKIM is only evaluated for _incoming_ mail (remember, inbound emails are checked, outbound emails are signed), the same goes for SPF, and the `policies` group is just a combination of both). So we just disable all checks and hence, we can remove the line about the re-writes.

@williamdes **can you please verify this configuration for me?**:) I will keep all checks enabled on my machine, and I cannot test it locally (simply lacking time).

Related to #3323

## Type of change

- [x] New feature (non-breaking change which adds functionality)
- [x] This change requires a documentation update

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation (README.md or the documentation under `docs/`)
- [x] If necessary I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

**Comment by georglauterbach** [comment]
<https://mcgrail.com> seems to be unreachable at the moment, hence the CI failures.

File: mailserver.env [hunk 1]
```diff
@@ -142,6 +142,13 @@ ENABLE_RSPAMD_REDIS=
 # 1     => enabled
 RSPAMD_LEARN=0
 
+# This settings controls whether checks should be performed on emails coming
+# from authenticated users (i.e. most likely outgoing emails). The default valu
```
Comment (williamdes) [inline mailserver.env:146]:
```suggestion
# from authenticated users (i.e. most likely outgoing emails). The default value
```

File: target/rspamd/local.d/settings.conf [hunk 2]
```diff
@@ -0,0 +1,12 @@
+# documentation: https://rspamd.com/doc/configuration/settings.html
+
+# DMS::SED_TAG::1::START
+# Disable all checks for authenticated users
+authenticated {
+    priority = high;
+    authenticated = yes;
+    apply {
+        groups_enabled = [];
```
Comment (williamdes) [inline target/rspamd/local.d/settings.conf:9]: This looks okay to me, I will try to test it in production. No promises but I will do my best 

Not sure if some kind of forwarding outbound flow could be affected

Comment (georglauterbach) [reply target/rspamd/local.d/settings.conf:9]: Nice. I'll merge when you sent me feedback about this :)

**Review by williamdes — APPROVED** [review]
Works fine in production !
See:
image

The score is 0 and there is no tokens in the list

**Review by casperklein — APPROVED** [review]

**Comment by georglauterbach** [comment]
> Works fine in production !

Noice! Then I will merge this :D Thank you @williamdes for your feedback!

**Pull request merged**